hugo.toml

```diff
@@ -0,0 +1,69 @@
+baseURL = 'https://tina.pm/'
+languageCode = 'en-us'
+title = "Tina's team website"
+theme = 'introduction'
+
+# For a configuration example of introduction theme, see
+# https://github.com/victoriadrake/hugo-theme-introduction/blob/master/exampleSite/config.toml
+
+DefaultContentLanguage           = "en"                    # Default language for multilingual sites
+
+[params]
```

Comment:
It took me a while to find the documentation for the settings in this section, as some core Hugo settings live in this same namespace, and these are theme settings. Would you add comments to separate theme-related settings from the rest?

Reply:
I think theme setting are thos that are in `params` sections. But maybe a reorder of config file could be applied.